backend/src/main/routes/AccountRoute.ts

```diff
@@ -40,51 +42,60 @@ export default class AccountRoute extends CommonRoutesConfig {
 
     this.getApp()
       .route(`/accounts/allEmployees`)
-      .get(async (req: express.Request, res: express.Response, next: express.NextFunction) => {
-        try {
-          const employeeAccounts = await this.employeeAccountService.getAllEmployeeAccounts();
-          if (employeeAccounts === null || employeeAccounts.length === 0) {
-            next(new HttpException(StatusCodes.NOT_FOUND, ReasonPhrases.NOT_FOUND));
-          } else {
-            const employeeAccountsDTOs: Array<any> = [];
-            employeeAccounts?.forEach((employeeAccount) => {
-              const dto = JSON.parse(JSON.stringify(employeeAccount));
-              delete dto.account.password;
-              employeeAccountsDTOs.push(dto);
-            });
-            res.status(StatusCodes.OK).send(employeeAccountsDTOs);
+      .get(
+        checkJwt,
+        checkRole(new Set([Roles.BUSINESS])),
```

Comment:
This endpoint should be only allowed for admin. Since it is retrieving all employees from the db.

Reply:
Fixed